### DISCUSSION TITLE
Idea: All possible utility classes applicable via typed prop

### DISCUSSION THREAD

**Discussion by jpandersen87** [MetroStar/comet#74] [Ideas]
It would be useful for all the possible utility classes (not `usa-` classes, as that's expected to be the component's responsibility and can be given via `className` by the user otherwise) to be applicable via a prop object similar to how the native `style` prop works.

**Comment by jbouder** [comment]
Easy access to utility classes would definite be super helpful. As for providing something like style, are you thinking something like this?

`<SomeComponent utilClass={{ widthFull: true, textBold: true }} /> `

**Reply by jpandersen87** [reply]
Yep, or if you're feeling extra daring simplify token-repeated utilities (ex: `{font: {sans: "lg"}} = font-sans-lg`) but a simple flat list as a start would be a huge accessibility gain.

**Reply by jbouder** [reply]
Sounds like a good idea to me. Created the following Issue for this here: https://github.com/MetroStar/comet/issues/77. Any chance you might be able to contribute to this idea?
